CoffeeChat/Controller/DemographicsViewController.swift

```diff
@@ -103,11 +103,19 @@ class DemographicsViewController: UIViewController {
     }
 
     @objc private func nextButtonPressed() {
-        /// Save onboarding information to user defaults, for user creation later.
-        for (key, value) in fieldValues {
-            userDefaults.set(value, forKey: key)
+        if let graduationYear = fieldValues[fieldMap[0]],
+           let major = fieldValues[fieldMap[1]],
+           let hometown = fieldValues[fieldMap[2]],
+           let pronouns = fieldValues[fieldMap[3]] {
+            NetworkManager.shared.updateUserDemographics(graduationYear: graduationYear,
+                                                  major: major,
+                                                  hometown: hometown,
+                                                  pronouns: pronouns,
+                                                  profilePictureURL: "").observe { successResponse in
+                                                    print("Update demographics success response \(successResponse)")
```

Comment:
also -- is this expected to work rn? when i run i get an error printed in the console 
```
Update demographics success response error(Swift.DecodingError.dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "Invalid value around character 0." UserInfo={NSDebugDescription=Invalid value around character 0.}))))
```

Reply:
the route got changed after I made PR! I changed the routes so now they work :)